### ISSUE TITLE
Integrity check fail 10.0.4RC1 on mimetype file

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#29277]
Install 10.0.4beta1 and check the integrity warnings.

The file core/js/mimetypelist.js must not be checked at all, but somehow a signature got generated for it.

It seems only half of the code was adjusted for exclusion. Also need to adjust the code that generates the signature.

This regression introduced through https://github.com/owncloud/core/pull/28973

**Comment by PVince81** [comment]
@VicDeo can you please have a look ?

**Comment by PVince81** [comment]
The issue still exists in beta2, reopening. Something else must be missing.

**Comment by VicDeo** [comment]
@PVince81 Let me guess. We sign all releases e.g. 10.0.4 beta2 with already installed owncloud that is NOT 10.0.4 beta2 and doesn't have changes from https://github.com/owncloud/core/pull/29316 ?

**Comment by PVince81** [comment]
Indeed. I've updated said instance to stable10, so let's recheck with tomorrow's daily.

**Comment by PVince81** [comment]
I built a temporary daily for testing and saw that the integrity check issues have disappeared.

**Comment by PVince81** [comment]
Still happening with 10.0.4 RC1.

@VicDeo please generate a dummy key and retest signing. It seems something must be going wrong and it is still signing the file, but excludes it later during some part of verification.

**Comment by VicDeo** [comment]
@PVince81 hmmm
```
> php occ status | grep version
  - version: 10.0.4.2
  - versionstring: 10.0.4 RC1

> php occ integr:sign-core --privateKey /develop/testkey/privateKey.key --certificate /develop/testkey/CSR.csr --path /develop/testkey/owncloud1004rc1/
Successfully signed "core"

> grep -R mimetypelist /develop/testkey/owncloud1004rc1/core/signature.json 
```

**Comment by PVince81** [comment]
Strange... when run from the release script it's in there.

If not reproducible I'll need to double check what the release script is doing.

**Comment by PVince81** [comment]
Arghh, my mistake... I used two workarounds that cancel each other out.

I'll do a test run tomorrow.

**Comment by PVince81** [comment]
solved in the 10.0.4 build
